clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1896,6 +1902,185 @@ private void recordRegionHybridConsumptionStats(
     }
   }
 
+  @Override
+  protected boolean isHybridFollower(PartitionConsumptionState partitionConsumptionState) {
+    return isHybridMode() && (isDaVinciClient || partitionConsumptionState.getLeaderFollowerState().equals(STANDBY));
+  }
+
+  /**
+   * Checks whether the lag is acceptable for hybrid stores
+   * <p>
+   * If the instance is a standby or DaVinciClient: Also check if <br>
+   * 1. the feature is enabled and is AA enabled: In non AA stores, consumer task is unable to read HB messages from RT
+   *           topic (though a test consumer can still read it), leading to standby replicas waiting for leader completion
```

Comment:
I thought the problem was the leader complete state would be jumping among UNKNOWN, INCOMPLETE and COMPLETE randomly, but seems like that's not the case from this comment. Why is consumer not able to read HB from RT?